server/render/index.js

```diff
@@ -180,6 +180,7 @@ export function serverRender( req, res ) {
 		context.isRTL = isLocaleRTL !== null ? isLocaleRTL : context.isRTL;
 	}
 
+	context.section = getSection( context.store.getState() );
```

Comment:
This code is executed on server. Only once, on Calypso initial load. That means that the initial section (the URL where the user entered Calypso) will be set on the `body` element forever. Won't change after navigating to another section.

Reply:
I think that the `context` already always has a `sectionName` property. So this code that adds `context.section` is not needed.

@blowery reports that he sees `.section-undefined` somewhere. That's strange -- `sectionName` should always be a valid correct string, no matter whether the section is isomorphic or not.